log/development.log

```diff
@@ -0,0 +1,2 @@
+  [1m[35m (720.2ms)[0m  [1m[35mCREATE DATABASE "team_2_grog_bot_api_development" ENCODING = 'unicode'[0m
```

Comment:
please put these logs in `.gitignore`

after that you need to remove chache locally, commit and push it up.
https://devconnected.com/how-to-clear-git-cache/

Reply:
updated that and cleared the cache